sdk/libs-scala/crypto/src/test/scala/com/daml/crypto/MessageSignaturePrototypeSpec.scala

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2025 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.crypto
+
+import java.security.{InvalidKeyException, KeyPairGenerator, SignatureException}
+import java.security.spec.ECGenParameterSpec
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import java.nio.charset.StandardCharsets
+
+class MessageSignaturePrototypeSpec extends AnyFlatSpec with Matchers {
```

Comment:
there's a lot of overlap between this test and MessageSignatureTest, do you intend to deduplicate them?

Reply:
Done